src/Microsoft.Sbom.Api/Workflows/SBOMParserBasedValidationWorkflow.cs

```diff
@@ -171,6 +171,13 @@ public async Task<bool> RunAsync()
                 recorder.RecordException(e);
                 log.Error("Encountered an error while validating the drop.");
                 log.Error($"Error details: {e.Message}");
+
+                if (e is FileNotFoundException || e is DirectoryNotFoundException)
+                {
+                    // Bubble up exception if user provided an invalid path/SBOM wasn't found
+                    throw;
```

Comment:
My guess is that this will break the existing behavior. Did you test your changes calling sbom-tool and compare the behavior before and after? Maybe the validation should happen within the new API method and not down here to avoid changing the existing behavior